docs/development/plugins/how-to.md

```diff
@@ -55,13 +60,11 @@ Let's create a plugin that just gets the number of pods in the cluster and
 displays that information in the top bar (i.e. registers an "app bar action").
 
 ```tsx
-import { Plugin } from '@kinvolk/headlamp-plugin/types/plugin/index.d';
-import Registry from '@kinvolk/headlamp-plugin/types/plugin/registry.d';
+import { Headlamp, Plugin, Registry } from '@kinvolk/headlamp-plugin/lib';
 
-const pluginLib = window.pluginLib;
-const React = window.pluginLib.React;
-const K8s = pluginLib.K8s.ResourceClasses;
-const { Typography } = pluginLib.MuiCore;
+// import { SectionBox } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+// import { K8s } from '@kinvolk/headlamp-plugin/lib/K8s';
+// import { Typography } from '@material-ui/core';
```

Comment:
Again, we should have the lines not commented out, and add only the ones needed.

Reply:
These were left commented out in the examples to make it easier for people to find where to import things from.